docs/dev/metrics.md

```diff
@@ -88,6 +88,28 @@ In addition, Prometheus provides some default metrics about the internal state
 of the running process and the metric collection. You can find more information
 about these metric names and their labels through the following links:
 
+## Machine API error rate for provider
+
+These metrics should help you collect and show errors, returned from the cloud API.
+
+Such metrics represent the cause for:
+- invalid provider configurations yet unknown to MAO static validation
+- timeouts and errors on the server side
+
+In addition they are counting error occurence over a defined timespan.
```

Comment:
Not sure what you're trying to say here, do you mean that these can be chopped up in prometheus to look at errors over a given period? What kind of metric is this? A counter?

Reply:
Just a note to the fact, that we initially planned those to be used for API failure rate. Yes, it is a counter.

Reply:
Ok, I'd drop this sentence, the rate etc can all be calculated/extrapolated in prometheus from any metric really